public/app/features/explore/RawList.tsx

```diff
@@ -0,0 +1,92 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { useCopyToClipboard } from 'react-use';
+
+import { GrafanaTheme } from '@grafana/data/src';
+import { IconButton, useStyles } from '@grafana/ui/src';
+
+import { instantQueryRawVirtualizedListData } from './RawListContainer';
+import RawListItem from './RawListItem';
+
+interface RawListProps {
+  listItemData: instantQueryRawVirtualizedListData;
+  listKey: number;
+}
+
+export type RawListValue = { key: string; value: string };
+
+const getStyles = (theme: GrafanaTheme) => ({
+  rowWrapper: css`
+    border-bottom: 1px solid #ccc;
```

Comment:
Is there some theming color that replaces `#ccc` raw declaration? I believe we have something like `theme.colors.border...`

Reply:
Fixed!